Client/Frontend/Browser/URLBarView.swift

```diff
@@ -37,28 +37,52 @@ class URLBarView: UIView, BrowserLocationViewDelegate, UITextFieldDelegate {
 
     private var locationView: BrowserLocationView!
     private var editTextField: ToolbarTextField!
-    private var locationContainer: UIView!
-    private var tabsButton: UIButton!
+    var locationContainer: UIView!
+    var tabsButton: UIButton!
     private var progressBar: UIProgressView!
-    private var cancelButton: UIButton!
+    var cancelButton: UIButton!
     private var curveShape: CurveView!
 
+    weak var browserToolbarDelegate: BrowserToolbarDelegate?
+
+    var shareButton: UIButton
+    var bookmarkButton: UIButton
+    var forwardButton: UIButton
+    var backButton: UIButton
+    var helper: BrowserToolbarHelper?
+    var showToolbar = false
```

Comment:
Nit: `showingToolbar`? `toolbarIsShown`?

`showToolbar` is an action, and sounds like a function.